app/web/components/Map.tsx

```diff
@@ -84,9 +84,6 @@ export default function Map({
   useEffect(() => {
```

Comment:
Map search page issues:

1. The map flickers when it first loads
2. The map flickers when you click on "Show XX on the map"
3. I'm unable to click on pins on the map! Only mouse interaction I can do on the map itself is panning.
4. Clicking on "Search this area" causes the "Ooops... that's an unknown error" screen!
5. Searching for a location also causes the oopsie screen

Reply:
Should be fixed now.